Command archive removal

usebeforefree opened the pull request:
Optimizing node performance. The main goals of this PR are:

- Removing command archiving

- Optimizing command removal by calling only one DB query, instead of two

usebeforefree committed:
updated cleaner command with findAndDeleteRows func

usebeforefree committed:
changed REPOSITORY_ROWS_FOR_REMOVAL to 50_000

usebeforefree committed:
ask-cleaner-command updated

usebeforefree committed:
ask-response-cleaner-command updated

usebeforefree committed:
blockchain-event-cleaner-command updated

usebeforefree committed:
commands-cleaner-command updated

usebeforefree committed:
finality-cleaner-command updated

usebeforefree committed:
get-cleaner-command updated

usebeforefree committed:
get-response-cleaner-command updated

usebeforefree committed:
publish-cleaner-command updated

usebeforefree committed:
publish-response-cleaner-command updated

usebeforefree committed:
update-cleaner-command updated

usebeforefree committed:
update-response-cleaner-command updated

usebeforefree committed:
removed everything command archiving related

usebeforefree committed:
removed everything command archiving related

usebeforefree committed:
Merge branch 'archive-removal' of github.com:marko03kostic/ot-node into archive-removal

usebeforefree committed:
permanent commands fix

usebeforefree committed:
Merge branch 'OriginTrail:v8/develop' into archive-removal

usebeforefree committed:
permanent commands updated

usebeforefree committed:
sql datetime fix

usebeforefree committed:
time delays updated

Mihajlo-Pavlovic commented on src/commands/cleaners/ask-cleaner-command.js:
```diff
@@ -16,8 +15,12 @@ class AskCleanerCommand extends CleanerCommand {
 
-    getArchiveFolderName() {
-        return ARCHIVE_ASK_FOLDER;
+    async findAndDeleteRows(nowTimestamp) {
```
This is nitpicking, but maybe just name function deleteRows

usebeforefree replied:
True, should I also remove the findRows and deleteRows funcs, since we don't use them anymore?

usebeforefree replied:
Done, also removed the obsolete findRows and deleteRows funcs.

Mihajlo-Pavlovic commented on src/commands/cleaners/update-cleaner-command.js:
```diff
@@ -31,7 +34,7 @@ class UpdateCleanerCommand extends CleanerCommand {
     default(map) {
         const command = {
-            name: 'publishCleanerCommand',
+            name: 'updateCleanerCommand',
```
Why did we change this?

usebeforefree replied:
It was a mistake, anyways the function is obsolete and not used anymore.

Mihajlo-Pavlovic commented on src/modules/repository/implementation/sequelize/repositories/blockchain-event-repository.js:
```diff
@@ -99,6 +99,18 @@ class BlockchainEventRepository {
+                processed: true,
+                createdAt: { [Sequelize.Op.lte]: timestamp },
+            },
+            order: [['createdAt', 'asc']],
```
Do we need this order by here?

usebeforefree replied:
Because the query is called in a loop until all rows that fit the criteria are deleted, it's safe to say we don't need it.

usebeforefree committed:
removed deleteRows and findRowsForRemoval

usebeforefree committed:
renamed findAndDeleteRows to deleteRows

usebeforefree committed:
removed obsolete order by from queries

usebeforefree committed:
query timestamp fixes

usebeforefree commented:
# Description

Cleaner commands updates:

Removed command archiving.
Optimized SQL queries by calling only delete, instead of find and then delete (2 calls).
Removed obsolete SQL queries.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

It has been tested on a local network. The tests consisted of passing in mock SQL data, which the cleaner commands should handle gracefully.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
